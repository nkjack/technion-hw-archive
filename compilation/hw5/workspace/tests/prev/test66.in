int Func(int x) {
	return x + 5;
}
void maxTestsCheckSumBetweenFuncs() {
	int x = Func(3) + Func(5);
	printi(Func(3));
	print("\n");
	printi(Func(5));
	print("\n");
	printi(x);
}

void main() {
     maxTestsCheckSumBetweenFuncs();
     print("\nI hate this course\n");
}
