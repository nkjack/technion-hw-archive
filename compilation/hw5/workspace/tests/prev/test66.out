Loaded: ./exceptions.s
8
10
18
I hate this course
